package-lock.json

```diff
@@ -1,7484 +1,8 @@
 {
   "name": "markdownlint-cli",
   "version": "0.30.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Not sure the package lock version should be downgraded, although it is "better" because Node 12 still ships with NPM 6

Reply:
Oh my apologies. Let me get my nvm up and running to be able to fix this. 

For everything else I do, I'm still using node 14 (which comes with NPM 6)

Reply:
@nschonni Ok take another look. I re-ran with node 16 and the diffs look much better